packages/sdk/src/interfaces/types.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  Provider,
+  TransactionReceipt,
+  TransactionResponse,
+} from '@ethersproject/abstract-provider'
+import { Signer } from '@ethersproject/abstract-signer'
+import { Contract, BigNumber } from 'ethers'
+
+
+/**
+ * L1 network chain IDs.
+ */
+export enum L1ChainID {
+  MAINNET = 1,
+  SEPOLIA = 11155111,
+  HARDHAT_LOCAL = 31337
+}
+
+/**
+ * Message finalizaton status.
+*/
+export enum MessageStatus {
+  // not ready to be finalized
+  PENDING = 0,
+  // ready to be finalized
+  READY = 1,
+  // already finalized
+  DONE = 2
+}
+
+/**
+ * L2 network chain IDs.
+ */
+export enum L2ChainID {
+  SPECULAR = 93481,
+  SPECULAR_HARDHAT_LOCAL = 31337,
```

Comment:
local testing l2 chain id is `13527` right?

Reply:
yes, we don't used this enum apparently